docs/devel/release-process.md

```diff
@@ -92,3 +99,59 @@ are issues that need to be handled before.
 If there is a security issue or an important bug fix we make patch releases in between. We don't
 have a formal definition of what an "important bug fix" is, it's up the team to discuss and decide
 whether to make a new release or not.
+
+## Making a bugfix release
+
+All newly merged patches fixing previous bug have to be backported to the latest stable release.
+When you merge a PR containing a patch fixing a bug, you should test if the buggy commits belongs to the latest release.
+Let's take an example with this commit:
```

Comment:
It's not clear to me, are we going to backport all fixes (even the very small ones) or are we going to check case by case if making a bug fix release for a specific commit is needed?

Reply:
I kind of agree, we should backport all commits marked with `Fixes:`, but we shouldn't mark all bug fixes to be backported. We can discuss on each PR what to do.